test/unit/fh3/cloud/test_cloud.js

```diff
@@ -16,9 +16,10 @@ module.exports = {
     });
   },
   tearDown : function(cb){
+    require('nock').cleanAll();
```

Comment:
nock will be a singleton anyway so I don't think you need to do the require in here. Can be at the top of the file like all others.